### PR TITLE
Release v0.0.6 to main

### PR DESCRIPTION
**Pull request by jsonsivar**
### Features
* Ethereum support (Goerli)
* contracts: ens integration
* contracts: verifier field and modifier
* contacts: prevent direct calls to implementation contract
* ui: simple create AP flow
* ui: navbar
* ui: form validations
* ui: explore view skeleton
* ui: explore view pagination
* ui: card list component
* ui: search on list
* ui: sort on list

### Fixes
* ui: color picker style
* ui: broken create ap button
* ui: graphql query (overlapping field)
* ui: colors not showing on firefox
* ui: alchemy keys missing

### Chores
* aws (serverless) project setup
* eslint cleanups
* contracts: change _billing to private
* ui: XYZ deployment
* ui: remove storybook
* ui: remove ens storage
* ui: remove unused refs
* ui: add base theme color black
* ui: drip stitches export
* ui: refactor style combobox
* ui: change mint forms fields order

### CODE REVIEW

**Review by EmperorOrokuSaki — CHANGES_REQUESTED** [review]
The subgraph tests are failing as expected. I suggest we remove all of them.
See #168

File: ui/src/components/core/color-picker/color-picker.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  const handleColorPickerClick = (): void => {
+    inputColorRef.current?.click();
+  };
+
+  return (
+    <Card.Text css={{ height: '$22', mt: '$6' }}>
+      <div className="relative">
+        <Flex css={{ gap: '$3h', alignItems: 'center' }}>
+          <span>Primary Color</span>
+          {/* TODO crate color picker component */}
```
Comment (csosamorales) [inline ui/src/components/core/color-picker/color-picker.tsx:44]: I'm gonna remove this comment for the next release

File: ui/src/components/core/input/input-file.styles.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    borderRadius: '$lg',
+
+    '&:hover': {
+      borderColor: '$gray8',
+    },
+
+    '&[aria-invalid=true], &[data-invalid]': {
+      borderColor: '$red9',
+    },
+    //TODO add error state
```
Comment (csosamorales) [inline ui/src/components/core/input/input-file.styles.ts:27]: will remove this also

**Review by csosamorales — APPROVED** [review]
Left some comments for the next release

**Review by EmperorOrokuSaki — APPROVED** [review]

**Review by zoruka — APPROVED** [review]

**Pull request merged**